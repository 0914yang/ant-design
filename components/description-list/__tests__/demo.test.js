import demoTest from '../../../tests/shared/demoTest';

demoTest('description-list');
