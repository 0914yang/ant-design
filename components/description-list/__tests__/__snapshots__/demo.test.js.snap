// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/description-list/demo/basic.md correctly 1`] = `
<div
  class="ant-description-list"
>
  <div
    class="ant-description-list-title"
  >
    User Info
  </div>
  <div
    class="ant-description-list-view"
  >
    <table>
      <tbody>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            UserName
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            Zhou Maomao
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Telephone
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            1810000000
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Live
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            Hangzhou, Zhejiang
          </td>
        </tr>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Remark
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            empty
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Address
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="3"
          >
            No. 18, Wantang Road, Xihu District, Hangzhou, Zhejiang, China
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;

exports[`renders ./components/description-list/demo/border.md correctly 1`] = `
<div
  class="ant-description-list border"
>
  <div
    class="ant-description-list-title"
  >
    User Info
  </div>
  <div
    class="ant-description-list-view"
  >
    <table>
      <tbody>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Product
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            Cloud Database
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Billing Mode
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            Prepaid
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Automatic Renewal
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            YES
          </td>
        </tr>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Order time
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            2018-04-24 18:00:00
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Usage Time
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="5"
          >
            2018-04-24 18:00:00 To 2019-04-24 18:00:00
          </td>
        </tr>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Status
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="5"
          >
            Running
          </td>
        </tr>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Negotiated Amount
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            $80.00
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Discount
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            $20.00
          </td>
          <td
            class="ant-description-list-item-label"
          >
            Official Receipts
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="1"
          >
            $60.00
          </td>
        </tr>
        <tr
          class="ant-description-list-item"
        >
          <td
            class="ant-description-list-item-label"
          >
            Config Info
          </td>
          <td
            class="ant-description-list-item-content"
            colspan="5"
          >
            Data disk type: MongoDB
            <br />
            Database version: 3.4
            <br />
            Package: dds.mongo.mid
            <br />
            Storage space: 10 GB
            <br />
            Replication_factor:3
            <br />
            Region: East China 1
            <br />
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
